assistant/src/main/java/com/ibm/watson/assistant/v1/model/UpdateDialogNode.java

```diff
@@ -0,0 +1,724 @@
+/*
```

Comment:
Why was this entire file created? Did it not exist before

Reply:
it's generated with the merge-patch

Reply:
```java
UpdateDialogNode updateDialogNode = new UpdateDialogNode.Builder()
              .description(dialogNodeDescription2)
              .nextStep(null)
              .dialogNode(dialogNodeName2)
              .build();
      Map<String, Object> body = updateDialogNode.asPatch();

      UpdateDialogNodeNullableOptions updateDialogNodeNullableOptions =
              new UpdateDialogNodeNullableOptions.Builder()
                      .workspaceId(workspaceId)
                      .dialogNode(dialogNodeName)
                      .body(body)
                      .build();
```
UpdateDialogNode contains the parameters that you can change/update
UpdateDialogNodeNullableOptions contains the parameters needed for the service call & the variables that were set in the UpdateDialogNode Model as a map

Reply:
Ok this makes sense. We will have to be clear in the documentation on how to use this new method